setup.py

```diff
@@ -74,13 +74,6 @@
     ],
     entry_points={
         "console_scripts": ["ape=ape._cli:cli"],
-        "ape_cli_subcommands": [
-            "ape_accounts=ape_accounts._cli:cli",
-            "ape_compile=ape_compile._cli:cli",
-            "ape_console=ape_console._cli:cli",
-            "ape_plugins=ape_plugins._cli:cli",
-            "ape_networks=ape_networks._cli:cli",
-        ],
```

Comment:
NOTE: The use of `entry_points` for these "1st party plugins" is also prototyping how 2nd/3rd party plugins would integrate into the CLI.

Reply:
If we go with `click.extensions`, we can have 2nd/3rd party plugins be installed via `pip` instead

Reply:
Would we be able to have the `ape_*` packages under `src` install in this way? would be nice to use the same mechanism for all of them.

Reply:
We can add `setup.py` to those as well!

Reply:
I'm open to restructuring the repo to be a series of projects, I just wanted it to be all installed via 1 `setup.py`

Reply:
Ok, I can look into this. I am pretty sure it's possible to install local `setup.py`s from a `setup.py` but that might not be a good solution in the end.

Does this mean users will not be able to uninstall them, or do we still want that?

I am curious as to the benefit of this. I understand that a consistent plugin installation is nice, but these might as well get installed the easy way -- by being imported / packaged together.

Reply:
We don't have to do this change in this PR, I think this is okay for now.

> Does this mean users will not be able to uninstall them, or do we still want that?

Yeah, the "1st class plugins" should never be uninstallable, they're sort of the core set of functionality required for ape to do anything useful at all. But still, I wanted them to be "1st class" instead of directly into the Core package (e.g. `ape`) to 1) help prototype the plugin interfaces, and 2) create a distinction between the core APIs and management functions and the "core" CLI/plugins provided by installing the package

Reply:
Small point: I like having the CLI endpoints set by the plugin's name (which is set via package name) to prevent malicious plugins from overriding core plugin's functionality. As these plugins have access to sensitive data stored on disk or provide privileged access to signing keys, I think it's important to take a reasonable amount of care to prevent impact as much as we can.

Reply:
We should need to stress this.. Users can always use `pip` on their own without `ape plugins add`, which may cause these issues?

Reply:
Yeah, the idea wasn't necessarily to block people from using `pip`, but give a default endpoint that avoided some gotchas. Can always reconfigure manually with `pip`